SSH/run-tests.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Script to automate the SSH format test
+
+JOHN_BINARY="../run/john"
+if [[ "$#" -ge 1 ]]; then
+    JOHN_BINARY="$1"
+fi
+
+TO_JOHN="../run/ssh2john.py"
+if [[ "$#" -ge 2 ]]; then
+    TO_JOHN="$2"
+fi
+TMP_DIR=$(mktemp -d)
+mkdir -p "$TMP_DIR/key"
+
+wget https://github.com/openwall/john-samples/raw/refs/heads/main/SSH/aes-256-ctr/id_rsa-aes256-ctr                 -O "$TMP_DIR"/key/id_rsa-aes256-ctr
+wget https://github.com/openwall/john-samples/raw/refs/heads/main/SSH/openssh-new-format-keys/ssh-new-format-1.key  -O "$TMP_DIR"/key/1.key
+wget https://github.com/openwall/john-samples/raw/refs/heads/main/SSH/openssh-new-format-keys/ssh-new-format-2.key  -O "$TMP_DIR"/key/2.key
+wget https://github.com/openwall/john-samples/raw/refs/heads/main/SSH/openssh-new-format-keys/ssh-new-format-3.key  -O "$TMP_DIR"/key/3.key
+wget https://github.com/openwall/john-samples/raw/refs/heads/main/SSH/sample_ssh_private_keys/dsa_test.key          -O "$TMP_DIR"/key/dsa.key
+wget https://github.com/openwall/john-samples/raw/refs/heads/main/SSH/sample_ssh_private_keys/rsa_test.key          -O "$TMP_DIR"/key/rsa.key
+
+wget https://github.com/openwall/john-samples/raw/refs/heads/main/SSH/passwords.txt                                 -O "$TMP_DIR"/passwords.lst
```

Comment:
If this script is to be in this repo, then it should use local copies of these files, not do any `wget`.

Reply:
The problem with this approach is that what I want to avoid is exactly needing to clone john-samples (which is a large repository) if I want to run this test.

Anyway, I'm adding this possibility to the script.